system/indy-node-tests/test_upgrade.py

```diff
@@ -11,11 +11,10 @@
 
 
 # TODO dynamic install of old version to upgrade from
-@pytest.mark.skip(reason='INDY-2132, INDY-2125')
+# @pytest.mark.skip(reason='INDY-2132, INDY-2125')
```

Comment:
Why comment instead of deleting? Are we unsure that problem is fixed?

Reply:
I'll skip it back, this test is not for CD yet.